### PR TITLE
Update English-to-Spanish NMT Example [KerasNLP]

### PR DESCRIPTION
**Pull request by abheesht17**
Resolves https://github.com/keras-team/keras-nlp/issues/259 and https://github.com/keras-team/keras-nlp/issues/190
- BLEU
- Use KerasNLP's WordPiece Tokeniser

### CODE REVIEW

File: examples/nlp/neural_machine_translation_with_keras_nlp.py [hunk 1]
```diff
@@ -38,11 +38,14 @@
...
 
-Before we start implementing the pipeline, let's import all the libraries we need.
+Before we start implementing the pipeline, let's install and import all the
+libraries we need.
 """
 
 """shell
 pip install -q rouge-score
+pip uninstall -y keras-nlp
+pip install -q git+https://github.com/keras-team/keras-nlp.git@fc01b18125bb707a0de7e442a6a8e48c8fe913a3
```
Comment (fchollet) [inline examples/nlp/neural_machine_translation_with_keras_nlp.py:48]: This should really be avoided -- we should install the latest version from pip.

File: examples/nlp/neural_machine_translation_with_keras_nlp.py [hunk 2]
```diff
@@ -516,17 +512,29 @@ def token_probability_fn(decoder_input_tokens):
...
 
 There are many metrics which are used for text generation tasks. Here, to
-evaluate translations generated by our model, let's compute the ROUGE-1 and
-ROUGE-2 scores. Essentially, ROUGE-N is a score based on the number of common
-n-grams between the reference text and the generated text. ROUGE-1 and ROUGE-2
-use the number of common unigrams and bigrams, respectively.
+evaluate translations generated by our model, let's compute the ROUGE-N and
+BLEU metrics.
+
+ROUGE-N is a metric based on the number of common n-grams between the reference
```
Comment (fchollet) [inline examples/nlp/neural_machine_translation_with_keras_nlp.py:518]: N-grams

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: examples/nlp/neural_machine_translation_with_keras_nlp.py (hunk 1 above)
Comment (abheesht17) [reply examples/nlp/neural_machine_translation_with_keras_nlp.py:48]: @fchollet, the reason for doing this is that certain APIs used in this example are not yet a part of any KerasNLP release. That's why I am using a particular commit for installation.

Comment (fchollet) [reply examples/nlp/neural_machine_translation_with_keras_nlp.py:48]: I'd suggest so, yes.

**Pull request closed without merging**